ocs_ci/ocs/node.py

```diff
@@ -643,7 +643,15 @@ def get_compute_node_names(no_replace=False):
     """
     platform = config.ENV_DATA.get("platform").lower()
     compute_node_objs = get_nodes()
-    if platform in [constants.VSPHERE_PLATFORM, constants.AWS_PLATFORM]:
+
+    if config.ENV_DATA.get("enable_flexible_scaling"):
+        return [
+            compute_obj.get()["metadata"]["labels"][constants.HOSTNAME_LABEL].replace(
+                ".", "-"
```

Comment:
will this logic is same for all platforms since above logic skips all platform if we enable flexible scaling

Reply:
Yes, it is the same for all platforms.
IMO, the platform information should not be part of this function . It should simply return the compute nodes list with or without the '.' based on the argument and the calling function should check based on platform.